applications/asset_tracker/README.rst

```diff
@@ -149,6 +149,18 @@ Request Power Saving Mode (PSM)
 	To enable this mode, set ``CONFIG_POWER_OPTIMIZATION_ENABLE=y`` and then
 	set Switch 2 to the GND position.
 
+.. _lwm2m_carrier_support:
+
+LWM2M Carrier Library Support
+*****************************
+This application supports the |NCS| :ref:`lwm2m_carrier` library.
+
+To enable the LWM2M carrier library, add the following to your ``west build`` command:
+
+``-DOVERLAY_CONFIG=lwm2m_carrier_overlay.conf``
```

Comment:
Can this be done in SEGGER Embedded Studio as well, and if so, how? Or is the only way to do this in SES to set all the configuration options manually?

Reply:
I am not familiar enough with SES to know the answer.